### PR TITLE
deque axe webdriverio example created

### PR DESCRIPTION
**Pull request by kaochris**
Created a deque example that navigates to a few of the pages in sauce demo and captures the accessibility results on sauce.

**Note: the latest version of axe-core/webdriverio does not seem to be working with sauce. I have set the version to a working one - 4.2.1

### CODE REVIEW

File: webdriverio/webdriver/examples/deque/test/specs/cart.accessibility.spec.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {setTestContext} from "../helpers";
+import {LOGIN_USERS, PAGES} from "../configs/e2eConstants";
+
+describe('Swag Items Accessibility Spec', () => {
+
```
Comment (wswebcreation) [inline webdriverio/webdriver/examples/deque/test/specs/cart.accessibility.spec.ts:5]: nitpicking, but maybe remove this line break

File: webdriverio/webdriver/examples/deque/test/specs/login.accessibility.spec.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+describe('Accessibility Spec', () => {
+
```
Comment (wswebcreation) [inline webdriverio/webdriver/examples/deque/test/specs/login.accessibility.spec.ts:2]: nitpicking, but maybe remove this line break

File: webdriverio/webdriver/examples/deque/test/specs/swag.details.accessibility.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import {setTestContext} from "../helpers";
+import {LOGIN_USERS, PAGES} from "../configs/e2eConstants";
+
+describe('Swag Items Accessibility Spec', () => {
+
```
Comment (wswebcreation) [inline webdriverio/webdriver/examples/deque/test/specs/swag.details.accessibility.ts:5]: nitpicking, but maybe remove this line break

File: webdriverio/webdriver/examples/deque/test/specs/swag.items.accessibility.spec.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import {setTestContext} from "../helpers";
+import {LOGIN_USERS, PAGES} from "../configs/e2eConstants";
+
+describe('Swag Items Accessibility Spec', () => {
+
```
Comment (wswebcreation) [inline webdriverio/webdriver/examples/deque/test/specs/swag.items.accessibility.spec.ts:5]: nitpicking, but maybe remove this line break

File: webdriverio/webdriver/examples/deque/test/specs/swag.items.accessibility.spec.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    it('should accessibility scan the swag items page', async () => {
+        //Navigate to the page that you'd like to scan
+        await setTestContext({
+            user: LOGIN_USERS.STANDARD,
+            path: PAGES.SWAG_ITEMS
+        });
+        //Scans the page for accessibility and pushes results to sauce labs
+        await browser.getAxeResults();
+    });
+
```
Comment (wswebcreation) [inline webdriverio/webdriver/examples/deque/test/specs/swag.items.accessibility.spec.ts:15]: nitpicking, but maybe remove this line break

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Hi @kaochris 

Thanks for this PR, some small comments and a question to add this into the main readme [here](https://github.com/saucelabs-training/demo-js) `WebdriverIO for Desktop Web (Selenium, but officially WebDriver) > Examples`

Then we can merge it

File: webdriverio/webdriver/examples/deque/package.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  "license": "MIT",
+  "devDependencies": {
+    "@axe-core/webdriverio": "4.2.1",
+    "@wdio/cli": "^7.16.12",
+    "@wdio/local-runner": "^7.16.12",
+    "@wdio/mocha-framework": "^7.16.11",
+    "@wdio/sauce-service": "^7.16.12",
+    "@wdio/shared-store-service": "^7.16.12",
+    "@wdio/spec-reporter": "^7.16.11",
+    "chromedriver": "^98.0.1",
```
[the remark this answers is not in the document]
Comment (kaochris) [reply webdriverio/webdriver/examples/deque/package.json]: removed chromedriver

File: webdriverio/webdriver/examples/deque/test/specs/login.accessibility.spec.ts (hunk 2 above)
Comment (kaochris) [reply webdriverio/webdriver/examples/deque/test/specs/login.accessibility.spec.ts]: removed line break

File: webdriverio/webdriver/examples/deque/test/specs/swag.details.accessibility.ts (hunk 3 above)
Comment (kaochris) [reply webdriverio/webdriver/examples/deque/test/specs/swag.details.accessibility.ts]: removed line break

File: webdriverio/webdriver/examples/deque/test/specs/swag.items.accessibility.spec.ts (hunk 4 above)
Comment (kaochris) [reply webdriverio/webdriver/examples/deque/test/specs/swag.items.accessibility.spec.ts]: removed line break

File: webdriverio/webdriver/examples/deque/test/specs/swag.items.accessibility.spec.ts (hunk 5 above)
Comment (kaochris) [reply webdriverio/webdriver/examples/deque/test/specs/swag.items.accessibility.spec.ts]: removed line break

**Comment by kaochris** [comment]
I don't know why it's running front end performance but I think it's probably the front end performance yml that's picking up my pr

File: README.md [hunk 7]
```diff
@@ -61,6 +61,8 @@ Node.js based automation frameworks locally and or in Sauce Labs.
         ![WebdriverIO - Webdriver - Examples - W3C](https://github.com/saucelabs-training/demo-js/workflows/WebdriverIO%20-%20Webdriver%20-%20Examples%20-%20W3C/badge.svg)
       - [WebdriverIO with Typescript](webdriverio/webdriver/examples/typescript)\
         ![WebdriverIO - Webdriver - Examples - TS](https://github.com/saucelabs-training/demo-js/workflows/WebdriverIO%20-%20Webdriver%20-%20Examples%20-%20TS/badge.svg)
+      - [WebdriverIO with Deque Axe Accessibility](webdriverio/webdriver/examples/deque)\
+        ![WebdriverIO - Webdriver - Examples - Deque Axe Accessibility](https://github.com/saucelabs-training/demo-js/actions/workflows/wdio.webdriver.examples.deque.axe.yml/badge.svg)]
```
Comment (wswebcreation) [inline README.md:65]: I don't see the workflow badge in GH actions in this PR, so it's better to remove it here

```suggestion
```

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Hi @kaochris 

The reason why the performance one is triggered is because you've changed the `package-lock.json`-file for the performance one.

I have one small suggestion, if you approve then we can merge it

Thanks man!

File: README.md (hunk 7 above)
Comment (wswebcreation) [reply README.md:65]: Lol, you already merged the workflow file. Can you please remove that. We are not testing on the existence of the accessibility report so having the workflow file has no added value

Comment (kaochris) [reply README.md]: I checked out the performance package-lock.json into my branch from main to revert whatever weird changes I made. Sorry.

File: webdriverio/webdriver/examples/deque/package.json [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "@axe-core/webdriverio": "4.2.1",
+    "@wdio/cli": "^7.16.12",
+    "@wdio/local-runner": "^7.16.12",
+    "@wdio/mocha-framework": "^7.16.11",
+    "@wdio/sauce-service": "^7.16.12",
+    "@wdio/shared-store-service": "^7.16.12",
```
Comment (wswebcreation) [inline webdriverio/webdriver/examples/deque/package.json:21]:
```suggestion
```

**Review by wswebcreation — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (wswebcreation) [reply README.md]: NP my friend, looks good now. I've just remove one last service that I forgot to check, but it's ok now.

Thanks

**Pull request merged**